services/rest/paths/pick-idea/pick-idea.js

```diff
@@ -116,18 +120,19 @@ let isRegenerating = false;
 
 /**
  * Regenerates an idea for the current dice sides
+ * @param {string} focusGroup
+ * @param {string} topic
+ * @param {string} medium
```

Comment:
I think the types need to be `string[]`

Reply:
ah, top thanks! I'm not too familiar with this kind of type declaration.